### PR TITLE
Enabling Windows builds.

### PR DESCRIPTION
**Pull request by moorepants**
Just seeing if the windows build magically works now.

### CODE REVIEW

**Comment by jakirkham** [comment]
Need to [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) to enable Windows builds.

**Comment by moorepants** [comment]
Thanks.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
 
 build:
     number: 201
-    skip: true                # [win]
+    skip: true                # [win and not py35]
```
Comment (moorepants) [inline recipe/meta.yaml]: I forget why I had it skipping everything but Py 35 on Windows. Is this needed?

Comment (jakirkham) [inline recipe/meta.yaml]: Does it require C++11? If so, `vc14`/VS 2015 is the only acceptable target. Currently that is a function of support Python versions. So Python 3.5 ensure it only builds on that target.

Comment (moorepants) [inline recipe/meta.yaml]: Yes, this is building with C++11.

**Comment by moorepants** [comment]
These appveyor and travis builds seem to have been queued for a mighty long time. Is this normal?

**Comment by jakirkham** [comment]
Is it normal? No. However, we have a pretty long backlog at this point. So we appreciate your patience while we get caught back up.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipe/meta.yaml]: Ok, then it's doing the right thing.

**Comment by moorepants** [comment]
Windows fails with the same error as before when trying to run the tests:

```
BUILD END: simbody-3.5.3-vc14_201
TEST START: simbody-3.5.3-vc14_201
WARNING: conda-build appears to be out of date. You have version 1.21.14 but the
latest version is 2.0.3. Run
conda update -n root conda-build
to get the latest version.
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    simbody-3.5.3              |         vc14_201        12.6 MB  local
The following NEW packages will be INSTALLED:
    python:         3.5.2-2        conda-forge
    simbody:        3.5.3-vc14_201 local       [vc14]
    vs2015_runtime: 14.0.25420-0   conda-forge
(root) C:\conda\conda-bld\test-tmp_dir>call activate.bat C:\conda\envs\_test 
TESTS FAILED: simbody-3.5.3-vc14_201
Command exited with code 1
```

**Comment by moorepants** [comment]
Any suggestions on how to get more useful output from this error? I'm not quite sure where to start when debugging it.

**Comment by jakirkham** [comment]
Please make sure to re-render with `conda-smithy` version `1.3.3`. There were some fixes there. Though not sure they are related to issues here or not.

**Comment by moorepants** [comment]
This has already been re-rendered with 1.3.3.

**Comment by moorepants** [comment]
I take that back. Only the master branch had.

**Comment by moorepants** [comment]
Trying removing the Windows file checks, which are:

```
   - if not exist %LIBRARY_BIN%\\simbody\\simbody-visualizer exit 1  # [win]
    - if not exist %LIBRARY_INC%\\simbody\\Simbody.h exit 1           # [win]
    - if not exist %LIBRARY_INC%\\simbody\\SimTKcommon.h exit 1       # [win]
    - if not exist %LIBRARY_INC%\\simbody\\SimTKlapack.h exit 1       # [win]
    - if not exist %LIBRARY_INC%\\simbody\\SimTKmath.h exit 1         # [win]
    - if not exist %LIBRARY_INC%\\simbody\\SimTKsimbody_aux.h exit 1  # [win]
    - if not exist %LIBRARY_INC%\\simbody\\SimTKsimbody.h exit 1      # [win]

    - if not exist %LIBRARY_LIB%\\libSimTKcommon.dll exit 1           # [win]
    - if not exist %LIBRARY_LIB%\\libSimTKmath.dll exit 1             # [win]
    - if not exist %LIBRARY_LIB%\\libSimTKsimbody.dll exit 1          # [win]
```

**Comment by moorepants** [comment]
Ok, my error is likely that the tests actually failed when trying to find these files on Windows, as it seems they are installed in different directories than I had originally figured out.

```
  -- Installing: C:/conda/envs/_build/Library/bin/simbody-visualizer.exe
  -- Installing: C:/conda/envs/_build/Library/include/Simbody.h
  -- Installing: C:/conda/envs/_build/Library/include/SimTKcommon.h
  -- Installing: C:/conda/envs/_build/Library/include/SimTKlapack.h
  -- Installing: C:/conda/envs/_build/Library/include/SimTKmath.h
  -- Installing: C:/conda/envs/_build/Library/include/SimTKsimbody_aux.h
  -- Installing: C:/conda/envs/_build/Library/include/SimTKsimbody.h

  -- Installing: C:/conda/envs/_build/Library/bin/SimTKcommon.dll
  -- Installing: C:/conda/envs/_build/Library/bin/SimTKmath.dll
  -- Installing: C:/conda/envs/_build/Library/bin/SimTKsimbody.dll
```

**Comment by jakirkham** [comment]
> This has already been re-rendered with 1.3.3.

So re-rendering is not only necessary for updates. It is also necessary when changing the `skip` conditions or when changing `yum_requirements.txt` as examples.

**Comment by moorepants** [comment]
What is an easy way to display on appveyor where things are installed to? My issue here seems to be that I don't have the correct paths to the files in the test section of the recipe, but I don't know how to find out exactly where they are installed on appveyor.

**Comment by isuruf** [comment]
7642a96 was correct except for the dll names `SimTKcommon.dll` instead of `libSimTKcommon.dll`

**Comment by moorepants** [comment]
Thanks, I've updated them.

**Comment by moorepants** [comment]
Woohoo! Finally passes. Merging.

**Pull request merged**